Project_One/dog.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <head>
+            <link rel="stylesheet" type="text/css" href="style.css">
+            <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+        </head>
+       
+        <button id = "grading-element-three" onclick="Show()">Show me the Dog</button>
+        
+        <div id="grading-element-three-consequence" >
+            <img class="animal" src="Photos/Dog.jpg">
+        </div>
+
+                        
+        <script>
+            function Show() {
+                var dog = document.getElementById("grading-element-three-consequence");
+                if (dog.style.display == "none") {
```

Comment:
almost works as expected. can you make it start as `visibility: hidden;` then run  `dog.style.visibility = 'visible'`? And then can you move it to `index.html`?